ext/CreateParametersExt.jl

```diff
@@ -51,4 +54,52 @@ function LandParameters(toml_dict::CP.AbstractTOMLDict)
     )
 end
 
+# TODO: Should this be organized into multiple files?
+"""
+    AutotrophicRespirationParameters(FT; kwargs...)
+    AutotrophicRespirationParameters(toml_dict; kwargs...)
+    AutotrophicRespirationParameters(;
+        ne,
+        ηsl,
+        σl,
+        μr,
+        μs,
+        f1,
+        f2,
+    )
```

Comment:
Is this constructor meant to be in this docstring?

Reply:
Actually it probably is good to document what's included in AutotrophicRespirationParameters, so users know what they can overwrite, but this sort of makes it look like a third constructor option which could be misleading

Reply:
Or maybe it's better for users to just look at `AutotrophicRespirationParameters` in src/standalone/Vegetation/autotrophic_respiration.jl if they want to see what they can overwrite

Reply:
It's probably better to remove it - as you said, users can look at the fieldnames if they want to circumvent these constructors.